src/scripts/main.js

```diff
@@ -1 +1,83 @@
 'use strict';
+
+const firstPromise = new Promise((resolve, reject) => {
+  const click = document.addEventListener('click', () => {
```

Comment:
click is always true, create a separated variable and set the true value only on click event